.github/workflows/contract-tests.yml

```diff
@@ -110,72 +110,80 @@ jobs:
           files: ./contracts/coverage.json
           verbose: false
           token: ${{ secrets.CODECOV_TOKEN }}
-  test-4844:
-    name: 4844 tests
-    runs-on: linux-2xl
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: recursive
-
-      - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1
-        with:
-          version: nightly
-
-      - uses: Layr-Labs/arbitrum-actions/run-nitro-test-node@main
-        with:
-          args: --pos
-          no-token-bridge: true
-
-      - name: Setup nodejs
-        uses: actions/setup-node@v2
-        with:
-          node-version: 18
-          cache: 'yarn'
-          cache-dependency-path: '**/yarn.lock'
-
-      - name: Install dependencies
-        run: yarn install
-
-      - name: Build
-        run: yarn build:all
-
-      - name: Test 4844
-        run: yarn test:4844
-  test-e2e:
-    name: Test e2e
-    runs-on: linux-2xl
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          submodules: recursive
-
-      - uses: Layr-Labs/arbitrum-actions/run-nitro-test-node@main
-        with:
-          l3-node: true
-          no-token-bridge: true
-          no-l3-token-bridge: true
-          nitro-contracts-branch: '${{ github.event.pull_request.head.sha || github.sha }}'
-
-      - name: Setup node/yarn
-        uses: actions/setup-node@v3
-        with:
-          node-version: 18
-          cache: 'yarn'
-          cache-dependency-path: '**/yarn.lock'
-
-      - name: Install packages
-        run: yarn
-
-      - name: Compile contracts
-        run: yarn build
-
-      - name: Run e2e tests
-        run: yarn test:e2e
-  test-e2e-custom-fee-token:
+  
+  # TODO: This is test is failing due testnode referencing a latest target when running
+  # a prysm ETH validator. This causes config ingestion to fail and the resource to fail to spin up.
+  # This has already been batched upstream and will need to be incorporated within the next rebasing.
```

Comment:
"batched" 

Did you mean "patched"?

Reply:
[2d95b36](https://github.com/Layr-Labs/nitro-contracts/pull/40/commits/2d95b369e91021820881550e079be9ccab5ee7a1)